--- a/configure
+++ b/configure
@@ -6025,7 +6025,7 @@

 case "$COMPONENTS" in
        all)
-               BUILDDIRS="tools filter backend berkeley cgi-bin monitor notifier ppdc scheduler systemv conf data desktop locale man doc examples templates"
+               BUILDDIRS="tools filter backend berkeley cgi-bin monitor ppdc scheduler systemv conf data desktop locale doc examples templates"
                ;;

        core)
@@ -6155,7 +6155,7 @@
        case "$host_os_name" in
                linux*)
                        if test -d /usr/lib64 -a ! -d /usr/lib64/fakeroot; then
-                               libdir="$exec_prefix/lib64"
+                               libdir="$exec_prefix/lib"
                        fi
                        ;;
        esac
